cmd/mistryd/job.go

```diff
@@ -118,6 +118,8 @@ func NewJob(project string, params types.Params, group string, cfg *Config) (*Jo
 	j.Image = ImgCntPrefix + j.ID
 	j.Container = ImgCntPrefix + j.ID
 
+	j.StartedAt = time.Now()
```

Comment:
`StartedAt` is not needed in `Job{}`. It should only exist in `BuildInfo{}`.

Reply:
True, we only need this for the index sort. Nevertheless, I say we take care of this in another PR as it affects the server and the job JSON encoding. Opened a new issue to track it https://github.com/skroutz/mistry/issues/49.